### PR TITLE
Improved muscle passive fiber force reporting

### LINKED ISSUE #835: PassiveFiberForceAlongTendon reported by MuscleAnalysis for Millard2012EquilibriumMuscle can be negative
Is this a bug?

I observed this with OpenSim 3.2 using the MuscleAnalysis. Perhaps it's due to small numerical inconsistencies resulting from using an approximate state, but even so, it seems like we shouldn't allow passive force to be negative.

I've attached the PassiveFiberForceAlongTendon.sto file as well as the files necessary to reproduce the issue.
[negative_passive_fiber_force_along_tendon.zip](https://github.com/opensim-org/opensim-core/files/174619/negative_passive_fiber_force_along_tendon.zip)

### PR DESCRIPTION
**Pull request by tkuchida**
Millard2012EquilibriumMuscle includes both conservative (elastic) and non-conservative (damping) terms when reporting passive fiber force. This PR improves the documentation for `Muscle::getPassiveFiberForce()` and `Muscle::getPassiveFiberForceAlongTendon()`, and adds outputs to Millard2012EquilibriumMuscle for reporting elastic and damping terms separately. Fixes #835 (satisfies points :one:, :two:, and :three:).

### CODE REVIEW

File: OpenSim/Actuators/Millard2012EquilibriumMuscle.h [hunk 1]
```diff
@@ -320,6 +332,17 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Millard2012EquilibriumMuscle, Muscle);
...
+    /** get the passive elastic force in the fiber (N) */
+    double getPassiveFiberElasticForce(const SimTK::State& s) const;
+    /** get the passive elastic force in the fiber (N) projected onto the tendon
+        direction */
+    double getPassiveFiberElasticForceAlongTendon(const SimTK::State& s) const;
+    /** get the passive damping force in the fiber (N) */
+    double getPassiveFiberDampingForce(const SimTK::State& s) const;
+    /** get the passive damping force in the fiber (N) projected onto the tendon
+        direction */
+    double getPassiveFiberDampingForceAlongTendon(const SimTK::State& s) const;
```
Comment (chrisdembia) [inline OpenSim/Actuators/Millard2012EquilibriumMuscle.h]: If I didn't have additional background, these doxygen comments probably would not be sufficient for me. What about:

```
get the portion of the passive fiber force coming from the damping element only
```

and similar for the elastic element.

**Comment by chrisdembia** [comment]
I made just one small comment. Appveyor builds failed because of update issues (#1167). I restarted the PR build, which should now pass because #1167 is now merged. We can ignore the branch build.

**Comment by tkuchida** [comment]
> I made just one small comment.

Clarified doxygen and merged master. Thanks for reviewing.

**Comment by chrisdembia** [comment]
LGTM.

**Comment by jenhicks** [comment]
This looks good to me, as well.

**Comment by tkuchida** [comment]
Test failure on Travis is familiar [testContactGeometry](https://travis-ci.org/opensim-org/opensim-core/jobs/151355931#L1960) timeout in Debug mode; already mentioned in #393.

**Pull request merged**